src/Style/Sass/layout/_navigation.scss

```diff
@@ -126,10 +126,89 @@
     position: relative;
     left: 0vw;
   }
+
+  #web-name {
+    display: none;
+  }
 }
 
 @media (min-width: 320px) {
   .nav-link {
     left: 90% !important;
   }
 }
+
+.profile-img {
+  cursor: pointer;
+  &-notify {
+    border-style: outset;
+    border-style: solid;
+    border-width: medium;
+    border-color: red;
+  }
+}
+
+.notify-container {
+  position: relative;
+  display: inline-block;
+  margin-top: 10px;
+}
+
+.notify-bubble {
+  position: absolute;
+  top: 0px;
+  right: 0%;
+  padding: 2px 5px 2px 6px;
+  background-color: red;
+  color: white;
+  font-size: 0.65em !important;
+  border-radius: 50%;
+}
+
+@media (max-width: 768px) {
+  .navbar-toggler {
+    border-color: white !important;
+  }
+  .navbar-toggler-icon {
+    border-color: white !important;
+  }
+  .notify-bubble {
+    right: 45.5%;
+  }
+
+  .notify-container {
+    text-align: center;
+  }
+
+  .nav-item-custom {
+    text-align: center;
+  }
+
+  .dropdown-menu-custom {
+    text-align: center;
+    width: 100%;
+    color: white;
+    background-color: #343a40;
+  }
+
+  .dropdown-item-custom {
+    color: white;
+    /* text-align: center; */
```

Comment:
commented codes

Reply:
Try to remove commented code and  try to  fix hound issues

Reply:
Thanks guys